vms/proposervm/vm.go

```diff
@@ -306,6 +306,11 @@ func (vm *VM) SetState(ctx context.Context, newState snow.State) error {
 func (vm *VM) BuildBlock(ctx context.Context) (snowman.Block, error) {
 	preferredBlock, err := vm.getBlock(ctx, vm.preferred)
 	if err != nil {
+		vm.ctx.Log.Error("unexpected build block failure",
+			zap.String("reason", "failed to fetch preferred block"),
+			zap.Stringer("parentID", vm.preferred),
```

Comment:
```suggestion
			zap.Stringer("preferredBlockID", vm.preferred),
```

Reply:
I wanted this to be `parentID` - because this is the block that we are trying to build on top of (the parent block) and all the other logs used `parentID`.